recipe_scrapers/__init__.py

```diff
@@ -629,41 +635,79 @@ def scraper_exists_for(url_path: str) -> bool:
 
 
 def scrape_html(
-    html: str, org_url: str | None = None, **options: dict[str, Any]
+    url: str,
+    html: str | None,
```

Comment:
Thanks for the review @jknndy!

Although I hope this is nearly ready, I'm concerned that one seemingly small change here (re-ordering `html` and `url` arguments... and renaming from `org_url`) could cause migration mayhem.

Perhaps there is a better way to do this to retain backwards-compatibility.

Reply:
I've restored the previous argument ordering.  It does appear that downstream callers use the `org_url` argument by name also.. so I guess it's worth keeping the same as `v14` too.

Reply:
Hrm.  I'm making more changes than I should after an existing code-review, but.. I think I'd like to make the arguments that should _always_ have an `=` (`online=...`, ...) into [keyword-only parameters](https://docs.python.org/3/glossary.html#term-parameter).

...and maybe provide a backwards-compatible, deprecated `wild_mode` param.

Reply:
Ok, I don't have any further changes planned for this pull request (in honesty, that's what I was beginning to think a few hours ago - but now I'm more confident that it's ready for an `rc2` release).